packages/components/src/components/button/button.scss

```diff
@@ -271,6 +271,19 @@
         animation-fill-mode: forwards;
         display: block;
     }
+    /* postcss-bem-linter: ignore */
+    .initial-loader--btn {
+        background-color: unset;
+
+        /* postcss-bem-linter: ignore */
+        .initial-loader__barspinner--rect {
+            background-color: $COLOR_WHITE;
+        }
+        /* postcss-bem-linter: ignore */
+        .barspinner {
+            margin: 0 4px 0 -4px;
+        }
+    }
```

Comment:
Comment: Having the `initial-loader--btn` class it outside of the `btn` class makes the `initial-loader` outside of the component supersede the level of importance so I'm nesting it inside the `btn` class and it seems like `bem` doesn't like it. 

So I'm just `postcss-bem-linter` ignoring it for now until we improve out class naming conventions for `initial-loader` component. I'd rather use this approach than use `!important`.